pkg/rehearse/jobs.go

```diff
@@ -276,6 +276,9 @@ func inlineCiOpConfig(container *v1.Container, ciopConfigs config.ByFilename, re
 	}
 	// inline CONFIG_SPEC for all ci-operator jobs
 	if container.Command != nil && container.Command[0] == "ci-operator" {
+		if !info.IsComplete() {
+			return fmt.Errorf("could not infer which ci-operator config this job uses")
```

Comment:
In general, it feels like it would be using what is missing, not only that something is missing

Reply:
Err, I'm not sure I get what this feedback suggests. Is the error message confusing? Or the logic? 

Adding this check actually prevents us from using something that is missing -- the `filename := info.Basename()` below returns a useless "--.yaml"-like filename.

Reply:
Which of org, repo or branch is empty

Reply:
oh, so you suggest that the error should contain what is missing?

Reply:
yes, exactly, I personaly would just make the `IsComplete` return an error that contains that info (or nil if everything exists)